packages/internals/src/get-generators/getGenerators.ts

```diff
@@ -495,3 +497,18 @@ In case you want to fix this, you can provide ${chalk.greenBright(
     }
   }
 }
+
+function filterGenerators(generators: GeneratorConfig[], generatorNames: string[]) {
+  if (generatorNames.length < 1) {
+    return generators
+  }
+
+  const filtered = generators.filter((generator) => generatorNames.includes(generator.name))
+
+  if (filtered.length !== generatorNames.length) {
+    const missings = generatorNames.filter((name) => filtered.find((generator) => generator.name === name) == null)
+    throw new Error(`The specified generator(s) ${chalk.bold(missings.join(', '))} do not exist in your Prisma schema`)
```

Comment:
This error message is only possible when `--generator` is used, right?
Then it could specifically mention "The generator `foo` specified via `--generator` does not exist in your Prisma schema" for example.

Additionally, differentiating between the singular and plural case by counting `missings` would be 👨‍🍳 💋 here.

Reply:
@janpio You're right. [e59dc48](https://github.com/prisma/prisma/pull/16452/commits/e59dc48d5aa03896d7d78deb3fe561872ade837a)